crt_portal/cts_forms/templates/forms/complaint_view/show/response_template.html

```diff
@@ -15,27 +15,12 @@ <h1>Contact complainant</h1>
         <p class="intake-template--description">
           <strong id="intake_description">(no template chosen)</strong>
         </p>
-        <select id="intake_select" class="usa-select" name="response_template" aria-label="select response template">
-          <option
-            value="none"
-            data-description="(no template chosen)"
-            data-content=""
-          >(select a response template)</option>
-          {% for template in templates %}
-            <option
-              value="{{ template.title | slugify }}"
-              data-description="{{ template.description }}"
-              data-content="{{ template.content }}"
-            >
-              {{ template.title }}
-            </option>
-          {% endfor %}
-        </select>
+        {{ templates }}
```

Comment:
Looks like the label `for` and the `select` ID here aren't quite aligned and throwing an a11y error. Same for the `textarea` field below.

```html
<label for="intake_select_0">Templates:</label>
<select name="templates" class="usa-select" id="intake_select">
```